README.md

```diff
@@ -52,6 +52,7 @@ All the common types used by the packages are defined [here](./common-types/inde
 - **Andrew Chen** - Developer
 - **Henry Li** - Developer
 - **Michael Ye** - Developer
+- **Samuel Yeboah** -Developer 
```

Comment:
Checks are failing because you forgot to put a space here between the dash and “Developer”

Reply:
thanks Andrew